frontend/src/component/TextInput.tsx

```diff
@@ -9,6 +9,7 @@ export enum TextInputType {
 interface TextInputProps {
   placeholder: string;
   type: TextInputType;
+  onChangeValue?: (newValue: string) => void;
```

Comment:
Revert changes in this file, merge main and use onChange directly. See Signup usage for an example.

Reply:
or revert and just leave as it is